Add Faker::TvShow namespace (#1431)
stympy/faker: doc/the_it_crowd.md

> # Faker::TheITTvShows::rowd

Author: ```suggestion
# Faker::TvShows::TheITCrowd
```

that's what I get for trying to copy-paste too fast

```ruby

> Faker::TheITvShows::Crowd.actor #=> "Chris O'Dowd"

Author: ```suggestion
Faker::TvShows::TheITCrowd.actor #=> "Chris O'Dowd"
```

> Faker::ThTvShows::ITCrowd.character #=> "Roy Trenneman"

Author: ```suggestion
Faker::TvShows::TheITCrowd.character #=> "Roy Trenneman"
```

> Faker::ThTvShows::ITCrowd.email #=> "roy.trenneman@reynholm.test"

Author: ```suggestion
Faker::TvShows::TheITCrowd.email #=> "roy.trenneman@reynholm.test"
```

> Faker::ThTvShows::ITCrowd.quote #=> "Hello, IT. Have you tried turning it off and on again?"

Author: ```suggestion
Faker::TvShows::TheITCrowd.quote #=> "Hello, IT. Have you tried turning it off and on again?"
```
